packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -323,24 +323,20 @@ export class FeatureDevController {
 
         getLogger().info(`${featureName} conversation id: ${session.conversationId}`)
 
+        // This is a loading animation
         this.messenger.sendAnswer({
-            type: 'answer',
+            type: 'answer-stream',
             tabID,
             message: approachCreation,
         })
 
-        // Ensure that the loading icon stays showing
-        this.messenger.sendAsyncEventProgress(tabID, true, undefined)
```

Comment:
@dianaabv16 we should only remove this and keep the `Placeholder` one as is.

Reply:
@dogusata done